src/main/java/at/aau/ase/cl/api/UserResource.java

```diff
@@ -39,4 +38,18 @@ public Response getUser(@PathParam("id") UUID id) {
         var result = UserMapper.INSTANCE.map(model);
         return Response.ok(result).build();
     }
+
+    @POST
+    @Path("user/{id}/address")
+    public Response addAddressToUser(@PathParam("id") UUID id, @Valid Address address) {
+        var modelAddress = AddressMapper.INSTANCE.map(address);
+
+        try{
+            var modelUser = service.addAddressToUser(id, modelAddress);
+            var result = UserMapper.INSTANCE.map(modelUser);
+            return Response.ok(result).build();
+        } catch (NotFoundException e) {
+            return Response.status(Response.Status.NOT_FOUND).entity("User with ID " + id + " not found").build();
```

Comment:
this catch will work, but it is not necessary, because `NotFoundException` from jax rs is translated into the not-found status code (404) out of the box.